lingui.config.ts

```diff
@@ -1,8 +1,60 @@
+import { default as babelExtractor } from '@lingui/cli/api/extractors/babel'
+import { createHash } from 'crypto'
+import { mkdirSync, readFileSync, writeFileSync } from 'fs'
+import * as path from 'path'
+import * as pkgUp from 'pkg-up'
```

Comment:
you might consider find-up instead, as it's the same functionality with more usage https://www.npmjs.com/package/find-up

Reply:
`pkgUp` is used internally by `lingui`, so I used it here so I could match the code from lingui's own extractors

Reply:
could you add this as a comment? useful info